modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
> a module that would both load data and match species before writing

yup, this is currently the case

> a second module that would match PFTs and then call veg2model

ah, I see, yes I kind of revert this one because it was two lines of code readRDS and match_pft to be called before veg2model.. thought they might as well be in the veg2model, and  I liked resembling veg2model to met2model by passing the path and prefix.. I can rearrange it if you prefer match_pft to be called outside